Library/Homebrew/test/support/fixtures/cask/Casks/with-login-items.rb

```diff
@@ -0,0 +1,9 @@
+cask "with-login-items" do
+  version "1.2.3"
+
+  url "file://#{TEST_FIXTURE_DIR}/cask/caffeine.zip"
+  homepage "https://brew.sh/"
+
+  app "Caffeine.app"
+  login_items "Caffeine.app"
```

Comment:
I think the dsl here should be `login_item`, and repeated if there are multiple.
This will keep it consistent with the other artifacts, that aren't pluralised.

Reply:
do you mean like how multiple `language` blocks are supported in the DSL?

Reply:
I'm referring to other artifacts such as `app`, `binary`, `artifact`, `font`, `qlplugin`, etc..

Reply:
If we look at the [everything.rb](https://github.com/Homebrew/brew/blob/0046519c08cd3967e0934c82195361ac7e76c567/Library/Homebrew/test/support/fixtures/cask/Casks/everything.rb#L47) DSL, there are other stanzas that are pluralized as well maybe?

Reply:
All of the artifact stanzas (which I think a login item will most closely relate to) are not pluralised.  

```

suite
app
pkg
installer
binary
manpage
colorpicker
dictionary
font
input_method
internet_plugin
keyboard_layout
prefpane
qlplugin
mdimporter
screen_saver
service
audio_unit_plugin
vst_plugin
vst3_plugin
```

For context, where multiple items doc the same type are allowed and provided the stanza is repeated rather than passing an array to the stanza.